gui-tk/wiring.tcl

```diff
@@ -497,9 +497,9 @@ proc canvasContext {x y X Y} {
     .wiring.context delete 0 end
     .wiring.context add command -label Help -command {help DesignCanvas}
     .wiring.context add command -label "Cut" -command cut
-    .wiring.context add command -label "Copy" -command minsky.copy
+    .wiring.context add command -label "Copy" -command "minsky.copy"
```

Comment:
Just for future reference, quotes are only needed when the command contains spaces.